README.md

```diff
@@ -12,8 +12,11 @@ So that's what Propshaft doesn't do. Here's what it does provide:
 
 ## Installation
 
-With Rails 7+, you can start a new application with propshaft using `rails new myapp -a propshaft` (pending the merge of [rails/rails#43261](https://github.com/rails/rails/pull/43261)).
+With Rails 7+, you can start a new application with propshaft using `rails new myapp -a propshaft`.
 
+## Upgrading from Sprockets
+
+Check the [upgrade guide](https://github.com/rails/propshaft/blob/master/UPGRADING.MD) for step-by-step instructions on how to upgrade an existing project to propshaft.
```

Comment:
Probably should be `main` and not `master` in this URL, btw.

Reply:
Thanks. Fixed.